x/launch/client/cli/tx_request_add_vested_account.go

```diff
@@ -0,0 +1,60 @@
+package cli
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	codec "github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/spf13/cobra"
+	"github.com/tendermint/spn/x/launch/types"
+)
+
+var _ = strconv.Itoa(0)
+
+func CmdRequestAddVestedAccount() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "request-add-vested-account [chainID] [vestingCoins] [vestingEndTime]",
```

Comment:
We must have two coin arguments:
- `initial-coins` (something like this) that is the initial amount of coins directly accessible to the account like a usual genesis account 
- `vesting-coins` the amount of coins vested at the vesting end time

Reply:
We can use `starting-balance` for the initial coins since this is the name of the field